modules/efs/main.tf

```diff
@@ -0,0 +1,60 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+resource "aws_kms_key" "encryption" {
```

Comment:
Might want to add a policy for the key so that its usage can be managed by IAM.

See what we do for the other keys in this repo.

Reply:
Okay added that in, default to root identifier as principal for all actions, but can be overridden.